api/src/main/java/io/strimzi/api/kafka/model/connect/AbstractKafkaConnectSpec.java

```diff
@@ -156,18 +155,8 @@ public void setJmxOptions(KafkaJmxOptions jmxOptions) {
         this.jmxOptions = jmxOptions;
     }
 
-    @Description("Metrics configuration. Only `jmxPrometheusExporter` can be configured, as this component does not yet support `strimziMetricsReporter`.")
+    @Description("Metrics configuration.")
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    @CelValidation(rules = {
-        @CelValidation.CelValidationRule(
-                rule = "self.type != 'jmxPrometheusExporter' || has(self.valueFrom)",
-                message = "valueFrom property is required"
-            ),
-        @CelValidation.CelValidationRule(
-                rule = "self.type != 'strimziMetricsReporter'",
-                message = "value type not supported"
-            )
-    })
```

Comment:
can you remember me why are we removing this CEL validation?

Reply:
@ppatierno AKAIU, We removed the rule becasue:

There are now two types of Metrics Reporters now so the rule around `!= 'jmxPrometheusExporter'` is obsolete. 

The logic around which type of `spec.getMetricsConfig()` is an instanceOf is handled in the model.

The type  `strimziMetricsReporter` IS now supported but it's value is optional.

Does this answer your question?